src/ApplicationLogs/LogReader.cs

```diff
@@ -23,31 +21,24 @@ public class LogReader : Plugin, IRpcPlugin, IPersistencePlugin
         public LogReader()
         {
             db = DB.Open(GetFullPath(Settings.Default.Path), new Options { CreateIfMissing = true });
+            RpcServer.RegisterMethods(this);
```

Comment:
ApplicationLogs is using RpcServer as references. Is it related to this? Loading sequence issue?

Reply:
I'm not able to pas ```CheckAuth```.
![image](https://user-images.githubusercontent.com/38493437/70596838-cdc59d00-1c21-11ea-9122-5a164e73e24a.png)

Reply:
Did you add `Authorization` entry in the HTTP headers?

Reply:
> Did you add `Authorization` entry in the HTTP headers?

Yes, Of course.
![image](https://user-images.githubusercontent.com/38493437/70597762-2302ae00-1c24-11ea-9880-b54680035db2.png)
![image](https://user-images.githubusercontent.com/38493437/70597698-f9498700-1c23-11ea-9994-c5b8cd2c5753.png)
![image](https://user-images.githubusercontent.com/38493437/70597739-15e5bf00-1c24-11ea-81fc-35ee8cc48419.png)
You can see they're the same.

Reply:
Then it should return `true`.

Reply:
Did you open wallet via RPC? Opening it in CLI has no effect. Only wallets opened in RPC can be accessed by RPC.

Reply:
You need to use `openwallet` command to open the wallet in RPC.

Reply:
Yes, but that's not the point. The issue is that `Settings.Default.RpcUser` and `Settings.Default.RpcPass` can't get value because of `Settings` conflict on ApplicationgLogs and RpcServer. So it can't return `true` here on authvalues[].
![image](https://user-images.githubusercontent.com/38493437/70600041-a541a100-1c29-11ea-8149-4cbe30f9417b.png)

Reply:
> You need to use `openwallet` command to open the wallet in RPC.

Yes, that's the truth. It works. But I'm a little confused that why wallet has to be opened by RPC to continue invoking other wallet related RPC?  Will it bring risks?

Reply:
There is no conflict at the runtime. Because they are both `internal` class. The conflict is in your debugger. You can't use the immediate window to get the values.